core/src/Instance/AadAuthority.cs

```diff
@@ -114,5 +102,10 @@ internal static bool IsInTrustedHostList(string host)
                 !string.IsNullOrEmpty(
                     TrustedHostList.FirstOrDefault(a => string.Compare(host, a, StringComparison.OrdinalIgnoreCase) == 0));
         }
+
+        internal override async Task Init(RequestContext requestContext)
+        {
+            await UpdateCanonicalAuthorityAsync(requestContext).ConfigureAwait(false);
```

Comment:
How will users of this class know they have to call Init?

Reply:
removed Init